sql/core/src/test/resources/sql-tests/inputs/transform.sql

```diff
@@ -342,3 +354,41 @@ SELECT TRANSFORM(b, MAX(a) AS max_a, CAST(sum(c) AS STRING))
 FROM script_trans
 WHERE a <= 2
 GROUP BY b;
+
+-- SPARK-33985: TRANSFORM with CLUSTER BY/ORDER BY/SORT BY
+FROM (
```

Comment:
can we use the standard TRANSFORM syntax? Or we write the tests for hive compatibility?

Reply:
> can we use the standard TRANSFORM syntax? Or we write the tests for hive compatibility?

Yea, for compatibility, since we have less test about transform before. Since I am always checking transform use case between hive and spark.

Reply:
Are MAP and REDUCE really compatible between Spark and Hive?

Reply:
> Are MAP and REDUCE really compatible between Spark and Hive?

Remove this two case.